docs/generators/changelog/versions/2.14.md

```diff
@@ -0,0 +1,193 @@
+## v2.14.0, 2023-10-XX
+
+### Highlights
+
+Hacktoberfest release! 🎉
+
+* MongoDB driver has been updated to v4.17.0
+
+* You can now set `DISABLE_SOCKJS_CORS=1` if you want to prevent SockJS from setting CORS headers. Do not set this option if you will have DDP clients from other origins connecting to the DDP server. [PR #12789]
+
+* Added guide on [how to prepare for Meteor 3.0 migration](https://guide.meteor.com/prepare-meteor-3.0).
+
+* New DDP merge strategy `NO_MERGE_MULTI`, which is similar to `NO_MERGE`, but it does track whether a document is used by multiple publications. [PR #12742]
+
+* Appcache has been further deprecated and moved to the deprecated packages folder.
+
+* Added `Accounts.createUserAsync` into the client.
+
+* Many packages had their underscore dependency removed.
+
+* Cordova has been updated to v12.0.1 for Android and v7.0.1 for iOS
+
+* `meteor create` command is now interactive!
+
+#### Migration Steps
+
+##### Android splash screen
+If you have been using `splash-screen` for Cordova, you need to update your code as Android changed their splash screen API, 
+the `cordova-plugin-splashscreen` is now on `cordova-android` core, so we have removed the dependency from the `splash-screen` 
+package. As a result we are dropping the support for dark mode splash screen on Android.
+
+To create this now you need to create two themes on your `config.xml` file.
+
+>  Note that it's still possible to have it by adding the according themes with App.appendToConfig and App.addResourceFile - but this is not something Meteor will do automagically right now.
+
+TODO: @matheusccastroo sample on how the new config should look and link to relevant Cordova docs 
```

Comment:
No, that is why we are still in beta.

@matheusccastroo can you please look at this.

Reply:
@StorytellerCZ I would recommend adding those two links here:
https://developer.android.com/develop/ui/views/launch/splash-screen#getting-started and https://developer.android.com/develop/ui/views/theming/darktheme.

Unfortunately I don't have a sample config on how it would look - I'm not using this option right now, so I'm also not sure how the config would look. From what I saw it is possible to be implemented, but it's not well documented on the cordova community, so it might not be too straightforward.

So I would say for now let's just point to the cordova/android docs.